conftest.py

```diff
@@ -764,3 +764,33 @@ def framework_agnostic(request, framework):
     if request.node.get_closest_marker("framework_agnostic"):
         if framework != default_framework:
             pytest.skip("framework agnostic test skipped for framework : {}".format(framework))
+
+
+@pytest.fixture
+def expected_values_amp(request):
+    """
```

Comment:
Is there any specific reason another expected_value fixture was created? Normally the existing `expected_values` fixture should work with any test. If for some reason, it does not work for this test, I think it would be better to alter the existing fixture as opposed to creating a new one. I can help out if the usage of the original `expected_values` fixture is unclear

Reply:
Solved.